extensions/src/main/java/org/pentaho/platform/web/http/filters/PentahoWebContextFilter.java

```diff
@@ -591,5 +682,9 @@ IPluginManager getPluginManager() {
   IUserSettingService getUserSettingsService() {
     return PentahoSystem.get( IUserSettingService.class, getSession() );
   }
+
+  ConfigurationAdmin getOSGIConfigurationAdmin() {
```

Comment:
This method won't be present in the next update of the PR

What I could find that for Java there is no single standard and it varies from case to case.
This is a good explanation -> https://stackoverflow.com/a/7366332

For other references I kept it like it was before: "...Osgi..."